lib/src/entrypoint.dart

```diff
@@ -245,13 +240,13 @@ class Entrypoint {
   /// Updates [lockFile] and [packageRoot] accordingly.
   Future<void> acquireDependencies(
     SolveType type, {
-    Iterable<String> unlock,
+    Iterable<String>? unlock,
```

Comment:
Could also be:
```suggestion
    Iterable<String> unlock = const [],
```
Not sure what I prefer....

Reply:
done